### PR TITLE
Add troubleshooting entry for inspecting artifact descriptors

### LINKED ISSUE #120: Document how to retrieve the descriptor of an artifact
We need to provide a guideline to tell users how to retrieve the descriptor of an artifact that is sent for signing. 

To retrieve the descriptor of an artifact from the remote registry, we can use `oras manifest fetch --descriptor`.

To retrieve the descriptor of a signature:
- COSE signatures: any CBOR inspector will help on reading the payload
- JWS signatures: to base64 decode the payload field of the JSON object

It should be documented on the website for troubleshooting purposes.

### PR DESCRIPTION
**Pull request by zr-msft**
Add troubleshooting entries for inspecting artifact descriptors.

fixes #120

### CODE REVIEW

File: content/en/docs/troubleshooting.md [hunk 1]
```diff
@@ -3,4 +3,26 @@ title: "Troubleshooting"
...
+Inspect the signatures of a remote artifact using `notation inspect` with the digest value for the artifact. For example:
+
+```console
+$ notation inspect IMAGE@sha256:111222333444555666777888999000aaaabbbbcccddddeeeeffff00011122233
+Inspecting all signatures for signed artifact
+IMAGE@sha256:111222333444555666777888999000aaaabbbbcccddddeeeeffff00011122233
+└── application/vnd.cncf.notary.signature
+```
+
+Confirm the expected signatures are present.
```
Comment (yizha1) [inline content/en/docs/troubleshooting.md:28]: Maybe we can add more details here for inspecting signatures .

For example, the output of `notation inspect` command contains
- certificate information, users can validate the signing certificates and certificate chain.
- signed artifact, which should contain the same descriptor information as the output from `oras manifest fetch`, to confirm the correct artifact is being signed.

**Review by yizha1 — COMMENTED** [review]
Suggest adding more information on the output of `notation inspect`

File: content/en/docs/troubleshooting.md (hunk 1 above)
Comment (zr-msft) [reply content/en/docs/troubleshooting.md]: thank you for the feedback @yizha1 

Updated the section to incorporate the additional details

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**